src/python/bot/fuzzers/engine.py

```diff
@@ -60,15 +61,16 @@ def prepare(self, corpus_dir, target_path, build_dir):
     FuzzOptions object."""
     raise NotImplementedError
 
-  def fuzz(self, target_path, options, max_time):
+  def fuzz(self, target_path, options, reproducers_dir, max_time):
     """Run a fuzz session. Returns a Result."""
     raise NotImplementedError
 
   def reproduce(self, target_path, input_path, arguments, max_time):
     """Reproduce a crash given an input. Returns a Result."""
     raise NotImplementedError
 
-  def minimize_corpus(self, target_path, dirs, max_time):
+  def minimize_corpus(self, target_path, arguments, output_dir, input_dirs,
```

Comment:
Why is arguments needed?

Reply:
There might be arguments that are necessary to always be passed to the target. There's definitely room for improvement here though, will think more once I make corpus pruning use this.